server/spec/job_status_spec.rb

```diff
@@ -123,18 +123,18 @@
       job = @user.autoheal_org(@owner['key'])
       #make sure we see a job waiting to go
       joblist = @cp.list_jobs(@owner['key'])
-      expect(joblist.find { |j| j['id'] == job['id'] }['state']).to eq('QUEUED')
+      expect(joblist.find { |j| j['id'] == job['id'] }['state']).to eq('RUNNING')
 
       @user.cancel_job(job['id'])
       #make sure we see a job canceled
       joblist = @cp.list_jobs(@owner['key'])
-      expect(joblist.find { |j| j['id'] == job['id'] }['state']).to eq('CANCELED')
```

Comment:
As mentioned in IRC, let's hang on to the CANCELED state, since it doesn't fit into either of the two completion states from the request, and it was preexisting so we won't break clients.

Reply:
Done.